packages/app/src/flame/valibot/flameSchema.ts

```diff
@@ -0,0 +1,86 @@
+import * as v from 'valibot'
+import { TransformVariationSchema } from './variationSchema'
+import type { DrawMode } from '../drawMode'
+
+// default values and schema fallbacks
+const latestValibotVersion = '1.0'
+const flameDescriptorVersionDefault = latestValibotVersion
+export const backgroundColorDefault: [number, number, number] = [0, 0, 0]
+export const backgroundColorDefaultWhite: [number, number, number] = [1, 1, 1]
+const cameraDefault: { zoom: number; position: [number, number] } = {
+  zoom: 1,
+  position: [0, 0],
+}
+const renderSettingsDefult = {
+  exposure: 0.25,
+  skipIters: 20,
+  drawMode: 'light' as DrawMode,
+  backgroundColor: backgroundColorDefault,
+  camera: cameraDefault,
+}
+const metadataDefault = {
+  version: flameDescriptorVersionDefault,
+  author: 'unknown',
+}
+
+export const transformIdSchema = v.pipe(v.string(), v.brand('TransformId'))
+export const variationIdSchema = v.pipe(v.string(), v.brand('VariationId'))
+export const variationTypeSchema = v.pipe(v.string(), v.brand('VariationType'))
+export const DrawModeSchema = v.union([v.literal('light'), v.literal('paint')])
```

Comment:
Not sure about casing on schema, it is kind of a type, but perhaps we should lowercase it?

Reply:
Alrighty, will update all